daprdocs/content/en/developing-applications/ides/vscode/vscode-how-to-debug-multiple-dapr-apps.md

```diff
@@ -0,0 +1,173 @@
+---
+type: docs
+title: "How-To: Debug multiple Dapr applications"
+linkTitle: "How-To: Debug multiple Dapr applications"
+weight: 40000
+description:  "How to configure Dapr for multiple applications"
+---
+
+As your project grows you will experience a need to configure VS Code to handle multiple Dapr applications to debug your project. This guide will offer you a simple to follow explanation of the type of changes you will need
+in order to ensure that your debugging experience is consistent.
+
+To follow along with the breakdown of configuration features please setup the [Hello World Quickstart Project](https://github.com/dapr/quickstarts/tree/v1.0.0/hello-world) 
+
+## Step 1: Configure launch.json
+This file contains information regarding the configurations you run during the debugging process. For the Hello World project, you will have two applications running along side 2 Dapr instances.
+Each SDK supported will require its own tweaks for the launching of the program but each configuration will contain a Daprd run task and a Daprd stop task.
+
+
+
+### NodeJS
+
+```json
```

Comment:
Are these correct JSON snippets? Last time we reviewed they were incomplete. Can a user copy and paste these and they would work?